frontend/test/metabase/scenarios/question/question-management.cy.spec.js

```diff
@@ -58,14 +60,84 @@ describe("managing question from the question's details sidebar", () => {
               cy.findByText("foo");
             });
 
-            it("should be able to move the question (metabase#11719-2)", () => {
-              // cy.skipOn(user === "nodata");
-              openQuestionActions();
-              cy.findByTestId("move-button").click();
-              cy.findByText("My personal collection").click();
-              clickButton("Move");
-              assertOnRequest("updateQuestion");
-              cy.contains("37.65");
+            describe("move", () => {
+              it("should be able to move the question (metabase#11719-2)", () => {
+                // cy.skipOn(user === "nodata");
+                openNavigationSidebar();
+                navigationSidebar().within(() => {
+                  // Highlight "Our analytics"
+                  cy.findByText("Our analytics")
+                    .parents("li")
+                    .should("have.attr", "aria-selected", "true");
+                  cy.findByText("Your personal collection")
+                    .parents("li")
+                    .should("have.attr", "aria-selected", "false");
+                });
+
+                openQuestionActions();
+                cy.findByTestId("move-button").click();
+                cy.findByText("My personal collection").click();
+                clickButton("Move");
+                assertOnRequest("updateQuestion");
+                cy.contains("37.65");
+
+                cy.contains(
+                  `Question moved to ${getPersonalCollectionName(USERS[user])}`,
+                );
+
+                navigationSidebar().within(() => {
+                  // Highlight "Your personal collection" after move
+                  cy.findByText("Our analytics")
+                    .parents("li")
+                    .should("have.attr", "aria-selected", "false");
+                  cy.findByText("Your personal collection")
+                    .parents("li")
+                    .should("have.attr", "aria-selected", "true");
```

Comment:
Ensure the sidebar collection is highlighted correctly after moving questions/models.